chronicles/textlineserializer.nim

```diff
@@ -17,22 +26,47 @@ serializationFormat TextLineLog,
                     mimeType = "text/plain"
 
 #
-# Class creation
+# color support functions
 #
-proc init*(T: type TextLineWriter, stream: OutputStream): T =
-  result.stream = stream
+proc fgColor(writer: TextLineWriter, color: ForegroundColor, brightness: bool) =
```

Comment:
These will be better as templates (this will lead to less work for the C compiler and leaner run-time code)

Reply:
Done. And moved them to options.nim so that both serializers can use them.